helm/kitcaddy/values.yaml

```diff
@@ -15,17 +15,31 @@ netpol:
   enabled: true
 
 kitcaddy:
+  startupProbe:
+    failureThreshold: 5
+    httpGet:
+      path: /metrics
+      port: 80
+      scheme: HTTP
+    initialDelaySeconds: 50
```

Comment:
Jeg synes ikke jeg oplever at det er normalt at det tager så lang tid for Kitcaddy for at være klar. Normalt plejer Kitcaddy at starte op på meget kort tid (< 1sek). 

Kan det være en ting i Netic miljøet der gør at det tager så lang tid? Så synes jeg default skal være lavere, så det ikke går ud over andre anvendere af Kitcaddy.